noir-projects/noir-protocol-circuits/crates/types/src/abis/event_selector.nr

```diff
@@ -0,0 +1,70 @@
+use crate::utils::field::field_from_bytes;
+use dep::std::cmp::Eq;
+use crate::traits::{Serialize, Deserialize, FromField, ToField, Empty};
+
+global SELECTOR_SIZE = 4;
+
+struct EventSelector {
+    // 1st 4-bytes of abi-encoding of an event.
```

Comment:
Would make more explicit what exactly this is, e.g., is it the leftmost or rightmost bytes that are the first?